### DISCUSSION TITLE
Organic position and sponsored position

### DISCUSSION THREAD

**Discussion by utsavvasoya** [amzn/ads-advanced-tools-docs#115] [Q&A]
How can I get an organic and sponsored position in Amazon ads for the campaign?
Please give me an answer as soon as possible.

**Comment by elizafarley** [comment]
Hi @utsavvasoya, if you are looking for performance by placement for your ads, you can use the Placement report 

- For Sponsored Brands, you can request a campaign report with a placement segment. [More info](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v2/report-types#placement-reports)
- For Sponsored Products, you can request a campaign report grouped by placement. [More info](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types#group-by-campaignplacement)

Is this what you are looking for?
